src/ThreeEditor/components/Sidebar/properties/category/ParticleFilterConfiguration.tsx

```diff
@@ -0,0 +1,46 @@
+import { Object3D } from 'three';
+
+import { PARTICLE_TYPES } from '../../../../../types/Particle';
+import { useSmartWatchEditorState } from '../../../../../util/hooks/signals';
+import { SetValueCommand } from '../../../../js/commands/SetValueCommand';
+import { YaptideEditor } from '../../../../js/YaptideEditor';
+import { isParticleFilter, ParticleFilter } from '../../../../Simulation/Scoring/ParticleFilter';
+import { ParticleSelect } from '../../../Select/ParticleSelect';
+import { PropertiesCategory } from './PropertiesCategory';
+
+export function ParticleFilterConfiguration(props: { editor: YaptideEditor; object: Object3D }) {
+	const { object, editor } = props;
+
+	const { state: watchedObject } = useSmartWatchEditorState(editor, object as ParticleFilter);
```

Comment:
do we need to use smartwatcheditor and monitor 3d editor here ?

Reply:
This is how all other "configuration" elements keep track of state of related object. Currently I dont know if there is another way to do it, and I dont think there is a reason to do it differently.

Reply:
OK, then lets keep it